dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -115,6 +115,10 @@ RUN pip install azure-kusto-data==0.0.13 \
 RUN pip install wheel==0.33.6
 RUN pip install pytest-ansible==2.2.2
 
+# Apply 'become_user' patch for pytest-ansible
+RUN curl -fsSL https://github.com/ansible/pytest-ansible/commit/d33c025f070a9c870220a157cc5a999fda68de44.patch | \
+    patch /usr/local/lib/python2.7/dist-packages/pytest_ansible/module_dispatcher/v28.py
```

Comment:
suggest to grab the source code and use python setup.py install to install it.

Reply:
Changes have been made to build pytest-ansible from source using setup.py.